libc/src/__support/FPUtil/BasicOperations.h

```diff
@@ -320,7 +321,7 @@ totalorder(T x, T y) {
   StorageType x_u = x_bits.uintval();
   StorageType y_u = y_bits.uintval();
 
-  using signed_t = cpp::make_signed_t<StorageType>;
+  using signed_t = make_integral_or_big_int_signed_t<StorageType>;
```

Comment:
For some reason `make_integral_or_big_int_signed_t` was not used anywhere in LLVM libc. I'm guessing it used to be referenced by code that was changed.